interactive/views.py

```diff
@@ -100,6 +102,9 @@ def analysis_request_email(request, pk):
     }
     send_analysis_request_email(analysis_request.user.email, context)
 
+    analysis_request.complete_email_sent = datetime.now()
```

Comment:
Django's `timezone.now()` should be prefered for datetimes to get a datetime which is correctly timezone naive or aware based on how the site is configured (with `USE_TZ`).